architecture/core_concepts/routes.adoc

```diff
@@ -830,6 +831,52 @@ Setting a server-side timeout value for passthrough routes too low can cause
 WebSocket connections to timeout frequently on that route.
 ====
 
+[[whitelist]]
+== Route Specific IP Whitelists
+You can restrict access to a route to a select set of IP addresses by using the 
+*haproxy.router.openshift.io/ip_whitelist* annotation on the route. The whitelist is a space
```

Comment:
Why asterisks not backticks?  I'd have thought we wanted to typeset them in fixed width.